release_automation.sh

```diff
@@ -66,6 +66,11 @@ gh auth status >/dev/null 2>&1 || abort "Error: You are not logged into any GitH
 # Check that jq is installed
 command -v jq >/dev/null || abort "Error: jq must be installed."
 
+# Updating the local spec repos benefits both Gutenberg and WPiOS which use CocoaPods
+# This avoids the scenario where the script aborts due to a 'CocoaPods could not find compatible versions for pod' error.
+ohai "Update the spec repos located at ~/.cocoapods/repos"
+execute "pod" "repo" "update"
```

Comment:
Any thoughts on ^that^ @guarani ?

Reply:
Thanks for flagging, @mchowning. 

👋 @fluiddot to respond to your above question (sorry for not replying earlier): 

> I'm wondering if we should execute this with the bundle command to assure that we use the expected cocoapods version of WPiOS, wdyt?

I think that's valid but I'm not sure there's a right answer. `pod repo update` updates the global `~/.cocoapods/repos` repository, which I understand is shared by all CocoaPods versions wherever they are installed. This suggests to me that `repo update` should be executed using the global `pod` rather than a project-specific CocoaPod version (`bundle exec pod ...`) [found in the Gemfile](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/Gemfile#L4).

Also, while WPiOS uses CocoaPods, so does Gutenberg, so if we don't use the global `pod` command, it's unclear if we should use [Gutenberg's version](https://github.com/WordPress/gutenberg/blob/ad766590e53a89b2c78ed5cbfe87dde2401699bd/packages/react-native-editor/ios/Gemfile#L3) (1.10.0) or [WPiOS' version](https://github.com/wordpress-mobile/WordPress-iOS/blob/2d53260aae667c67fae1e6f4c1f871a3a839ba02/Gemfile#L4) (1.11) of CocoaPods.

Reply:
Ok, thanks for the explanation 🙇 . I understand that the use of `bundle` is more important when installing the dependencies, as it might change the `COCOAPODS` value in `Podfile.lock`. Being this said, I'll consider my comment solved 👍 .

Reply:
Makes sense, thanks for raising this though 👍